PrivateCloud.DiagnosticInfo/PrivateCloud.DiagnosticInfo.psm1

```diff
@@ -1851,24 +1881,25 @@ function Get-SddcDiagnosticInfo
                 $null = Get-ClusterLog -Node $using:ClusterNodes.Name -Destination $using:Path -Health -UseLocalTime
             }
         }
-
-        $JobStatic += $($ClusterNodes).Name |% {
-
-            Start-Job -Name "System Info: $_" -ArgumentList $_,$ClusterDomain,$AccessNode -InitializationScript $CommonFunc {
-
-                param($NodeName,$DomainName,$AccessNode)
-
-                $Node = "$NodeName"
-                if ($ClusterDomain.Length) {
-                    $Node += ".$DomainName"
+			
+        $JobStatic += $ClusterNodes.Name |% {
+		
+			$NodeName = $_
+			
+            Invoke-SddcCommonCommand -ClusterNodes $AccessNode -JobName "System Info: $NodeName" -InitBlock $CommonFunc {

```

Comment:
And here is your double hop. All of the gathers here are natively remotable, and we use that. By remoting the script block to the AccessNode, you're stuck.